src/config/rcxml.c

```diff
@@ -442,12 +442,29 @@ entry(xmlNode *node, char *nodename, char *content)
 		rc.snap_edge_range = atoi(content);
 	} else if (!strcasecmp(nodename, "topMaximize.snapping")) {
 		rc.snap_top_maximize = get_bool(content);
+
+	/* <windowSwitcher show="" windowPreview="" windowOutlines="" /> */
+	} else if (!strcasecmp(nodename, "show.windowSwitcher.core")) {
+		rc.cycle_view_osd = get_bool(content);
+	} else if (!strcasecmp(nodename, "windowPreview.windowSwitcher.core")) {
+		rc.cycle_preview_contents = get_bool(content);
+	} else if (!strcasecmp(nodename, "windowOutlines.windowSwitcher.core")) {
```

Comment:
These checks include the `window` prefix.

Reply:
Whooa. Thanks for catching.